### PR TITLE
add data quality flag and parameter limits

### PR DESCRIPTION
**Pull request by ax-sc**
By adding a `data_quality_warning` flag to the measurement model, we are able to present a warning for measurements, which contain anomalous parameters. This allows us to be more precise in moderating data entered by users. It should be used for measurements, which follow the general rules (so there is no need the hide/delete them as a consequence) but have parameters with odd values and will only be set/unset by admins/moderators in the admin interface.

At the same time, (upper and lower) limits for parameter types are introduced to do a soft check for sanity of the parameter values and raising the `requires_review` flag when exceeding those limits.

### CODE REVIEW

**Comment by jnsdrtlf** [comment]
Thanks for the work on this. Did I understand correctly that `data_quality_warning` is generally only set by admins or moderators in the admin interface and not automatically set when a limit is exceeded? Currently only `requires_review` is set. I like and prefer that behaviour 👍🏽

File: gcampus/core/receivers.py
Comment (jnsdrtlf) [inline gcampus/core/receivers.py:122]: You can return after setting `requires_review = True` for the first time. The second case does not have to be checked and given that `upper_warning_limit` > `lower_warning_limit` it should never be true anyway.

**Review by jnsdrtlf — CHANGES_REQUESTED** [review]

**Comment by jnsdrtlf** [comment]
Only a small code-quality improvement but feel free to merge without a second review after fixing my only comment.

**Comment by jnsdrtlf** [comment]
Also: Can you rebase the branch on `main` to resolve the conflicts before merging`

File: gcampus/core/receivers.py
Comment (jnsdrtlf) [inline gcampus/core/receivers.py:131]: Last `return` is not required but doesn't bother me either.

**Review by jnsdrtlf — APPROVED** [review]

**Comment by ax-sc** [comment]
> Thanks for the work on this. Did I understand correctly that `data_quality_warning` is generally only set by admins or moderators in the admin interface and not automatically set when a limit is exceeded? Currently only `requires_review` is set. I like and prefer that behaviour 👍🏽

Thanks for the review and the feedback. The `data_quality_warning` field is exclusively set in the admin interface, yes.

**Pull request merged**
